services/src/test/scala/cromwell/services/database/LiquibaseComparisonSpec.scala

```diff
@@ -331,13 +340,25 @@ object LiquibaseComparisonSpec {
       HsqldbTypeTimestamp -> ColumnType("DATETIME"),
     ),
     defaultMapping = Map(
-      ColumnDefault(HsqldbTypeBoolean, Boolean.box(true)) ->
-        ColumnDefault(ColumnType("TINYINT", Option(3)), Int.box(1)),
+      HsqldbDefaultBooleanTrue -> Int.box(1)
     ),
   )
 
-  // MariaDB should behave exactly the same as MySQL
-  private val MariadbColumnMapping = MysqldbColumnMapping
+  // MariaDB should behave similar to MySQL except that only LOBs have sizes
+  private val MariadbColumnMapping =
+    ColumnMapping(
+      typeMapping = Map(
+        HsqldbTypeBigInt -> ColumnType("BIGINT"),
```

Comment:
The keys in the map are what HSQLDB calls the column. The values are what the other RDBMS calls the column. In this case MariaDB.

The way the matching works, at the start of the test an HSQLDB schema is created with Slick instead of Liquibase. Then one by one each RDBMS type is initialized, liquibased, and then the resulting schema is compared to the HSQLDB-Slick schema. Any mismatches in column type are an error. Except sometimes there are minor differences.

In this line HSQLDB has columns that are `ColumnType("BIGINT", Option(64))` while MariaDB doesn't return sizes for "BIGINT" so its equivalent column type is a `CoulmnType("BIGINT", None)`.